kie-camel/pom.xml

```diff
@@ -114,7 +114,6 @@
     <dependency>
       <groupId>org.kie</groupId>
       <artifactId>kie-spring</artifactId>
-      <scope>test</scope>
```

Comment:
I tried `mvn clean install -DskipTests -Dfull` with the test scope and it works. Can you please put it back to test? Otherwise we would have many artifacts on compile path which are not necessary.

Reply:
@MarianMacik It doesn't work for me. When I run `mvn clean install -DskipTests -Dfull` with the test scope the `maven-enforcer-plugin` complains about duplicate classes caused by `spring-jcl`. This happens even though I clearly exclude that dependency from `kie-spring`. Changing the scope back to compile fixes this. Could this be a bug in the enforcer in test scope?

Also, does it work when you run `mvn clean install -Dfull`? Don't we need to run tests to verify before merging?

Reply:
I'll try adding the test scope back and see what Jenkins says. Perhaps it's an issue with my local machine.